### PR TITLE
Add a widget to show how to use calcite components

### PR DESCRIPTION
**Pull request by ming8525**

### CODE REVIEW

File: widgets/use-calcite-components/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Use calcite components widget
+
+This sample demonstrates how to use [Calcite components](https://developers.arcgis.com/calcite-design-system/components/) in custom widget.
```
Comment (qlqllu) [inline widgets/use-calcite-components/README.md:3]: We'd better add something similar to the content in FAQ like this: before you use components in Calcite, we suggest you check components in jimu-ui...

Comment (mike-han) [reply widgets/use-calcite-components/README.md:3]: Where should we add these instructions?

Comment (qlqllu) [reply widgets/use-calcite-components/README.md:3]: @mikey you can find it here: https://faq-preview.sites.afd.arcgis.com/experience-builder/guide/frequently-asked-questions/#can-i-use-components-from-calcite-when-how

**Comment by gavinr** [comment]
I was trying to update the readme and somehow pushed too much to this branch. I've created a fixed branch/PR here that contains the same changes that was in this PR: #101

**Pull request closed without merging**